src/visualize/image_like.jl

```diff
@@ -127,35 +109,114 @@ function play{T}(buffer::Array{T, 2}, video_stream, t)
     return reinterpret(T, buffer, (w,h))
 end
 
-"""
-Takes a 3D image and decides if it is a volume or an animated Image.
-"""
-function _default{T<:Colorant, X}(img::Images.Image{T, 3, X}, s::Style, data::Dict)
-    props = img.properties
-    if haskey(props, "timedim")
-        if haskey(props, "spatialorder")
-            get!(data, :spatialorder, join(props["spatialorder"], ""))
+# If the user is using the new Images, ImageAxes will be loaded
+if isdefined(Images, :ImageAxes)
+    unwrap(img::Images.ImageMeta) = unwrap(data(img))
```

Comment:
I do think that GLAbstraction is the right place to make this kind of decision, and then to loosen the types on any higher-level functions. As a bonus, any extra information that gets "carried around" (unusual indices, pixel spacing, etc) by an AbstractArray type will be available until the very last possible moment, when you need to transfer a block of memory to the GPU.